torch/_inductor/lowering.py

```diff
@@ -7389,6 +7389,26 @@ def control_deps_op_lowering(additional_deps, subgraph_fn, *args):
     return output
 
 
+from torch._inductor.fx_passes.control_dependencies import requires_deps
+
+
+@register_lowering(requires_deps, type_promotion_kind=None)
+def requires_deps_op_lowering(additional_deps, out):
+    # Realize all additional dependencies
+    dep_names = []
+    for dep in additional_deps:
+        if not isinstance(dep, IRNode):
+            continue
+
+        dep.realize()
+        dep_names.append(dep.get_name())
+
+    for dep_name in dep_names:
+        V.graph.additional_buffer_deps[out.get_name()].add(dep_name)
```

Comment:
https://github.com/pytorch/pytorch/blob/a2413b9ced3bbd4193f776193461dd1a51dc5bfb/torch/_inductor/lowering.py#L7373

`additional_buffer_deps` is from operator name -> buffer deps. 

Here, we're doing buf -> buf. 

I'm not sure this is being applied